component-library/component-lib/styles.scss

```diff
@@ -1,6 +1,6 @@
 @use '~@ircc-ca/ds-sdc-core/index' as ircc-ds;
 @include ircc-ds.theme-init-required(ircc-ds.palette-journeylab(),
     default,
-    large,
+    small,
```

Comment:
Pretty sure this should still be 'large'

Reply:
Pushed all changes, thanks :)